pkgs/development/python-modules/reproject/default.nix

```
@@ -32,18 +32,21 @@ buildPythonPackage rec {
 
   postPatch = ''
     substituteInPlace pyproject.toml \
-      --replace "cython==" "cython>="
+      --replace "numpy>=2.0.0rc1" "numpy"
```

```suggestion
      --replace "numpy>=2.0.0rc1" "numpy" \
      --replace-fail '"--strict-markers", ' ""
```

A resolution for the failing reproject build is to remove strict-markers flag from pyproject.toml